server/channels/store/searchlayer/user_layer.go

```diff
@@ -135,14 +135,14 @@ func (s *SearchUserStore) autocompleteUsersInChannelByEngine(engine searchengine
 	if result.NErr != nil {
 		return nil, errors.Wrap(result.NErr, "failed to get user profiles by ids")
 	}
-	inUsers := result.Data.([]*model.User)
+	inUsers := result.Data
 	autocomplete.InChannel = inUsers
```

Comment:
I think we can simplify here
```suggestion
	autocomplete.InChannel = result.Data
```